src/app/calendar/_components/timetable/TimeTableMainPart.tsx

```diff
@@ -16,12 +19,16 @@ type weekdaysMapType = {
   Sun: Reservation[];
 };
 
-type ReservationWithTimestamp = {
+export type ReservationWithTimestamp = {
   reservation: Reservation | null;
   timestamp: number;
 };
 
-export function CalendarTimeTable({ reservations }: CalendarTimeTableProps) {
+export function TimeTableMainPart({
+  reservations,
+  typeFilters,
+  selectedRoom,
+}: TimeTableMainPartProps) {
   function getWeekDay(date: string) {
```

Comment:
this function can be outside of the component (you don't want to create it each time the component is rendered)

Reply:
Ok, I created separate utils file for them